tests/rptest/services/admin.py

```diff
@@ -533,6 +537,26 @@ def _request(self,
             self.redpanda.logger.debug(f"Dispatching {verb} {url}")
             try:
                 r = self._session.request(verb, url, **kwargs)
+                # Requests library does NOT respect Retry-After with a 307
```

Comment:
fold the above request into the loop below?

Reply:
Reworked this section of the code.